src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -12,14 +12,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
     <Reference Include="System.Configuration" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Fody" Version="2.1.2" PrivateAssets="All" />
```

Comment:
I guess we were getting Fody as a transitive dependency of `Janitor.Fody` and `Obsolete.Fody`. Is there a reason to add it as a top level dependency? If we dropped those two Fody extensions would we still need it?